src/pkg/services/m365/api/lists.go

```diff
@@ -11,6 +11,40 @@ import (
 	"github.com/alcionai/corso/src/pkg/services/m365/api/graph"
 )
 
+var legacyColumns = map[string]struct{}{
```

Comment:
I like your impulse to centralize these values.  It's good for code readability and cleanliness.  Looking at the usage of these maps, however, I think there's a couple extra protections that would be nice.  In general, this suggestion comes from how using global maps can produce a smell, since alterations to those maps at runtime can be difficult to detect and debug.

For the most part, it seems like you're using the maps purely as a check for whether a key exists in a certain set.  For those uses, I'd suggest getter funcs like `hasKey(string) bool`.

In one other case, you're actually looking for a slice of the keys, and are hacking the map to do the same.  In that case I'd suggest a  func like `keys() []string`.

Putting it together, you can easily get both cases with a type wrapper, like so:
```go
type keySet map[string]struct {}

func (ks keySet) hasKey(k string) bool {...}
func (ks keySet) keys() []string {...}

var legacyColumns = keySet{...}
```

That makes it safe and easy for implementations to do the same check you're currently doing by instead calling `if legacyColumns.hasKey(k) {...}`